src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const expensesService = require('../services/expenses.service');
+const userService = require('../services/users.service');
+
+const getAllExpenses = (req, res) => {
+  const expenses = expensesService.getAll(req.query);
+
+  res.json(expenses);
+};
+
+const getOneExpense = (req, res) => {
+  const { id } = req.params;
+
+  const expense = expensesService.getById(+id);
+
+  if (!expense) {
+    res.statusCode = 404;
+    res.statusMessage = 'Error';
+    res.end();
+  }
+  res.json(expense);
+};
+
+const createExpense = (req, res) => {
+  const { userId, spentAt, title, amount, category, note } = req.body;
```

Comment:
Consider wrapping the body of the function into the try-catch block and sending the 500 error if something goes wrong in the try block